### PR TITLE
Add "remove bot automerge" command

### PR DESCRIPTION
**Pull request by maresb**
@conda-forge/core, could you please give me a branch for testing?

Checklist
* [ ] Pushed the branch to main repo
* [ ] CI passed on the branch

### CODE REVIEW

File: conda_forge_webservices/commands.py [hunk 1]
```diff
@@ -393,13 +395,21 @@ def issue_comment(org_name, repo_name, issue_num, title, comment):
                 changed_anything |= make_rerender_dummy_commit(git_repo)
 
             elif ADD_BOT_AUTOMERGE.search(text):
-                pr_title = "[ci skip] [cf admin skip] ***NO_CI*** adding bot automerge"
```
Comment (beckermr) [inline conda_forge_webservices/commands.py:396]: You need to leave these other skip commands.

File: conda_forge_webservices/commands.py [hunk 2]
```diff
@@ -393,13 +395,21 @@ def issue_comment(org_name, repo_name, issue_num, title, comment):
...
+                pr_title = "[ci skip] adding bot automerge"
                 comment_msg = "added bot automerge"
                 to_close = ADD_BOT_AUTOMERGE.search(title)
                 check_bump_build = False
                 extra_msg = "\n\nMerge this PR to enable bot automerging.\n"
 
                 changed_anything |= add_bot_automerge(git_repo)
+            elif REMOVE_BOT_AUTOMERGE.search(text):
+                pr_title = "[ci skip] removing bot automerge"
```
Comment (beckermr) [inline conda_forge_webservices/commands.py:406]: Please add the skip commands from above here.

File: conda_forge_webservices/commands.py [hunk 3]
```diff
@@ -802,6 +813,36 @@ def add_bot_automerge(repo):
...
+    if len(cfg['bot']) == 0:
+        del cfg['bot']
+    with open(cf_yml, 'w') as fp:
+        yaml.dump(cfg, fp)
+
+    # now commit
+    repo.index.add([cf_yml])
+    author = Actor("conda-forge-admin", "pelson.pub+conda-forge@gmail.com")
+    repo.index.commit(
+        "[ci skip] ***NO_CI*** removed bot automerge", author=author)
```
Comment (beckermr) [inline conda_forge_webservices/commands.py:842]: You need the skip commands here I think.

Comment (beckermr) [reply conda_forge_webservices/commands.py:842]: Also it looks like I forgot them above so if you can add them up there that'd be good.

**Comment by maresb** [comment]
@beckermr, thanks, I think I got them all. Anything else?

**Comment by beckermr** [comment]
Not so far. I'll bring this change over to a branch for testing and let you know about anything else.

**Comment by beckermr** [comment]
Ahhh there is one thing. We'll need a pr on the docs repo adding documentation for this command. Lmk if you need help finding where to put it.

**Comment by maresb** [comment]
@beckermr, draft docs PR now at https://github.com/conda-forge/conda-forge.github.io/pull/1805

**Pull request merged**